tests/test_output_transformer.py

```diff
@@ -48,12 +48,12 @@ def test_output_transformer():
     query_result = runQuery(testquery)
 
     """ start test of output transformer """
-    qOutTrans = QueryOutputTransformer()
+    qOutTrans = QueryOutputTransformer(query_result)
```

Comment:
Here the results com from a custom query executioner function. For the future we should change this and have two different kinds of tests instead. One kind with results from the actual query executioner (functional test) and another kind of test with results that are simply hard coded for the tests cases (unit test). The former tests our package end to end a latter test the queryTransformer in isolation without dependencies on functions that are not part of the package.